deployment/data-feeds/changeset/aptos/deploy.go

```diff
@@ -0,0 +1,42 @@
+package aptos
+
+import (
+	"fmt"
+
+	"github.com/aptos-labs/aptos-go-sdk"
+
+	modulefeeds "github.com/smartcontractkit/chainlink-aptos/bindings/data_feeds"
+	cldf "github.com/smartcontractkit/chainlink-deployments-framework/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/data-feeds/changeset/types"
+)
+
+func DeployDataFeeds(chain cldf.AptosChain, owner aptos.AccountAddress, platform aptos.AccountAddress, secondaryPlatform aptos.AccountAddress, labels []string) (*types.DeployDataFeedsResponse, error) {
```

Comment:
```suggestion
func DeployDataFeeds(chain cldf_aptos.Chain, owner aptos.AccountAddress, platform aptos.AccountAddress, secondaryPlatform aptos.AccountAddress, labels []string) (*types.DeployDataFeedsResponse, error) {
```

please use `cldf_aptos.Chain` going forward, `cldf.AptosChain` will be removed soon